about.html

```diff
@@ -0,0 +1,311 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,700;0,900;1,400&display=swap" rel="stylesheet">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z"
+        crossorigin="anonymous">
+
+    <link rel="stylesheet" href="css/main.css">
+
+    <title>Capstone</title>
+</head>
+
+<body>
+    <div class="container-fluid">
+        <!--upper navigation-->
+
+
+        <div class="row top_navigation">
+            <div class="col-7"></div>
+            <div class="col-3">
+                <ul>
+                    <li>
+                        <img src="images/facebook.png" alt="facebook">
+                    </li>
+                    <li>
+                        <img src="images/twitter.png" alt="twitter">
+                    </li>
+                    <li>English</li>
+                    <li>My page</li>
+                    <li>Logout</li>
+                </ul>
+            </div>
+            <div class="col-2"></div>
+        </div>
```

Comment:
- [ Required ] Please consider aligning the social media icons to the right of the topbar section as illustrated in the behance design shown below.
You can do this by refactoring your code as shown below
```HTML
<header class="row top_navigation">
    <div class="col-9"></div>
       <div class="col-3">
          <ul>
             <li> <img src="images/facebook.png" alt="facebook"> </li>
             <li> <img src="images/twitter.png" alt="twitter"></li>
             <li>English</li>
             <li>My page</li>
             <li>Logout</li>
         </ul>
     </div>
</header>
```

## Behance Design
![Screenshot from 2023-01-12 16-04-24](https://user-images.githubusercontent.com/16392046/212074798-d48cfd58-be4e-4450-b47c-263d6c588445.png)

## Your implementation
![Screenshot from 2023-01-12 16-14-01](https://user-images.githubusercontent.com/16392046/212075872-3aff77b1-4431-4ccc-b1e1-89c0f858ed19.png)